R/tm_g_association.R

```diff
@@ -222,11 +222,31 @@ srv_tm_g_association <- function(id,
   with_filter <- !missing(filter_panel_api) && inherits(filter_panel_api, "FilterPanelAPI")
   checkmate::assert_class(data, "tdata")
   moduleServer(id, function(input, output, session) {
+    overlap_vars <- function(value) {
+      if (length(selector_list()$ref()$select) != 0 &&
+        selector_list()$ref()$select %in% selector_list()$vars()$select) {
+        "Associated variables and reference variable cannot overlap"
+      }
+    }
     selector_list <- teal.transform::data_extract_multiple_srv(
       data_extract = list(ref = ref, vars = vars),
-      datasets = data
+      datasets = data,
+      select_validation_rule = list(
+        ref = shinyvalidate::compose_rules(
+          shinyvalidate::sv_required("At least one reference variable needs to be selected."),
```

Comment:
You've put "at least one" but if you try to allow more then you get this:

![image](https://user-images.githubusercontent.com/15201933/209315408-0a3118fe-1b8a-4fa7-ac4a-e9c36a4c386c.png)

So I think this should say "A reference variable needs to be selected"

Reply:
updated.